randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -257,8 +256,13 @@ Extra - room_id: [6]
   * Layers: default
   > Door to Sciser Stall
       Any of the following:
-          # Destroy bomb blocks by standing off of crab: https://youtu.be/h8Q6TKHg8kM
-          Hi-Jump and Stand On Frozen Enemies (Beginner) and Can Freeze Enemies and Can Use Bombs and Can Use Springball
+          All of the following:
+              # Destroy bomb blocks by standing off of crab: https://youtu.be/h8Q6TKHg8kM
+              Stand On Frozen Enemies (Beginner) and Can Freeze Enemies and Can Use Bombs
+              Any of the following:
+                  Hi-Jump
```

Comment:
This may just be me, I'd rather have you use the template here for future-casing.